### ISSUE TITLE
Bypass blocks accessibility improvements

### ISSUE THREAD

**Issue by trebuan202** [gchq/stroom#5403]
2.4.1 Bypass Blocks (Partially supports)
Whilst Stroom provides a wide range of shortcuts to enable you to get direct access to different screens and to enable you to perform actions, there were some areas where you were unable to bypass repeated content. As the selectable menu options were located in the left panel you would use these to indicate your selection. In the case where there were existing items ( e.g. pipeline, XSL etc), any selected item would appear in the central panel. Using the keyboard, in order to move to the central panel to inspect the selected item one had to navigate through repeated content.
